cli/marathon.py

```diff
@@ -256,3 +259,75 @@ def getTasks(self, roger_env, environment):
         respjson = resp.json()
         tasks = resp.json()['tasks'] if 'tasks' in respjson else {}
         return tasks
+
+<<<<<<< HEAD
+    # def app_id(self, template_file):
+    #     """
+    #     returns the application id for the given template file
+    #
+    #     :params:
+    #     :template_file [str]: absoulte path to the template file
+    #     :return: [dict]
+    #     """
+    #     dir_name = os.path.dirname(template_file)  # returns the  directory name
+    #     file_name = os.path.basename(template_file)  # returns the basename
+    #     env = Environment(loader=FileSystemLoader(dir_name))  # loads filename using the dir_name
+    #     template = env.get_template(file_name)  # gets a template using a filename
+    #     return yaml.safe_load(str(template.module))["id"]  # returns the value of the id
+=======
+    def app_id(self, template_file):
+        """
+        returns the application id for the given template file
+
+        :params:
+        :template_file [str]: absoulte path to the template file
+        :return: [dict]
+        """
+        dir_name = os.path.dirname(template_file)
+        file_name = os.path.basename(template_file)
+        env = Environment(loader=FileSystemLoader(dir_name))
+        template = env.get_template(file_name)
+        return yaml.safe_load(str(template.module))["id"]
+>>>>>>> 363157802599f5896dc846705b64486d610e1313
+
+    def image_name(
```

Comment:
https://www.python.org/dev/peps/pep-0008/#function-names

Please see link and let me know your thoughts.